autobisect/main.py

```diff
@@ -99,9 +119,18 @@ def _parse_args(argv=None):
             parser.error('Detect mode set to log-limit but no limit set!')
         if args.detect == 'memory' and args.memory is None:
             parser.error('Detect mode set to log-limit but no limit set!')
-
-    if args.branch is None:
-        args.branch = 'central'
+    elif args.target == 'js':
+        if args.detect == 'diff':
+            if args.arg_1 is None or args.args_2 is None:
+                parser.error('Detect mode set to diff but no arguments supplied!')
+        if args.detect == 'hang':
+            if args.hang_time is None:
+                parser.error('Detect mode set to hang but no hang threshold supplied!')
+            if args.hang_time > args.timeout:
+                parser.error('Hang threshold greater than timeout!')
+        if args.detect == 'output':
+            if args.arg_1 is None or args.args_2 is None:
```

Comment:
Comparison to `None`

Reply:
What's wrong with that?

Reply:
Probably more Pythonic to use `if not` though I see it's already been changed to `.match`